readthedocs/search/tests/test_api.py

```diff
@@ -9,6 +11,7 @@
 
 @pytest.mark.django_db
 @pytest.mark.search
+@override_settings(ELASTICSEARCH_DSL_AUTOSYNC=True)
```

Comment:
This can be used with unittest only! we need to use pytest here

Reply:
So how do you do it with pytest?